docs/context_attributes.rst

```diff
@@ -35,6 +35,7 @@ stdout_capture  scenario  :class:`~StringIO.StringIO`                   If stdou
 stderr_capture  scenario  :class:`~StringIO.StringIO`                   If stderr  capture is enabled.
 table           step      :class:`~behave.model.Table`                  Contains step's table, otherwise None.
 text            step      String                                        Contains step's multi-line text (unicode), otherwise None.
+attach          step      attach(mime_type, data)                       Embed data in formatter output (currently only JSON formatter).
```

Comment:
While it's good to have this documented at least somehow, this information is now added to the list of "Behave Attributes". The `attach` is a member, though, a callable. It somehow doesn't fit.

@jenisys What would be a better way to document this?

Reply:
I documented the feature in a separate section in the formatters page instead, I think that's more suitable. I also added an example for embedding screenshots with selenium.